tests/src/main/java/fixtures/header/models/HeadersResponseEnumHeaders.java

```diff
@@ -12,14 +12,14 @@
      * The value property.
      */
     @JsonProperty(value = "value")
-    private GreyscaleColors value;
+    private Enum0 value;
```

Comment:
This might be a autorest.core issue.

Yesterday I updated to latest autorest-beta and autorest.core, regenerate header.json with our latest java v4. And this enum is  `GreyscaleColors` (I think this should be the correct class, not Enum0), and `Enum0.java` is not generated.